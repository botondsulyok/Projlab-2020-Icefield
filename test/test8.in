field stable 0
field unstable 1 false 0
neighbours stable unstable
neighbours unstable stable
entity mover Eskimo stable
entity puller Eskimo stable
stats stable
stats unstable
item rope1 rope puller
item ds1 suit mover
stats mover
stats puller
move mover unstable
stats mover
stats unstable
stats stable