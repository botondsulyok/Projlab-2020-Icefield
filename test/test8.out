id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: mover, puller
pullers: mover, puller
masters: mover, puller
neighbours: unstable

id: unstable
type: unstable
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable
capacity: 1
broken: false

id: mover
type: eskimo
iceUnit: stable
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: ds1
pullerDevice: basicpull0
tent: null
components: null

id: puller
type: eskimo
iceUnit: stable
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: rope1
tent: null
components: null

id: mover
type: eskimo
iceUnit: unstable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: ds1
pullerDevice: basicpull0
tent: null
components: null

id: unstable
type: unstable
building: plains4
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: stable
capacity: 1
broken: true

id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: puller
pullers: puller
masters: puller
neighbours: unstable

